persistence-utils-demo-test/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,11 +1,11 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=17
-org.eclipse.jdt.core.compiler.compliance=17
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=21
+org.eclipse.jdt.core.compiler.compliance=21

```

Comment:
you enforce java 21 ... thats why the build fails

Reply:
how did you craft this change? is it intended to comply with master?

Reply:
I used the convert function in the Ivy Designer v11.4 Nightly

Reply:
if you use the nightly, you will need java21... then the advice is to use temurin21.